custom_components/wyzeapi/light.py

```diff
@@ -196,22 +192,30 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                     self._bulb.effects = "3"
 
         _LOGGER.debug("Turning on light")
-        loop = asyncio.get_event_loop()
-        loop.create_task(self._bulb_service.turn_on(self._bulb, self._local_control, options))
```

Comment:
I'm not sure why these were being run in the event loop. Tested with lights both local and cloud control, no issues.